### DISCUSSION TITLE
How to use code from another repo without importing?

### DISCUSSION THREAD

**Discussion by ghost** [QuiteAFancyEmerald/InvisiProxy#127] [General]
This isn't exactly a question about the site but about Github itself. I was wondering how you guys managed to use the code from bunker without actually importing it. I am decently new to GitHub and seen this. It would be very helpful for the game site I am making. I looked online but it didn't make much sense to me. Would you guys mind explaining how?

**Comment by QuiteAFancyEmerald** [comment]
Are you asking how to clone repositories? Or referring to `.gitmodules`? (Which you can find in the root directory here)

Here is a good guide regarding its usage. :D 
https://git-scm.com/book/en/v2/Git-Tools-Submodules

**Reply by ghost** [reply]
If I go into here
https://github.com/QuiteAFancyEmerald/Holy-Unblocker/tree/master/views/archive
There is a bunker folder. It looks like you are using code from that repo without importing the code itself? How exactly did you do that? Is that what the link above is talking about?

**Reply by QuiteAFancyEmerald** [reply]
Yeah lol

**Reply by ghost** [reply]
Ok, thank you so much!

**Reply by QuiteAFancyEmerald** [reply]
ofc np :D
